ldap/servers/slapd/libglobs.c

```diff
@@ -827,6 +827,10 @@ static struct config_get_and_set
      NULL, 0,
      (void **)&global_slapdFrontendConfig.ldapi_search_base_dn,
      CONFIG_STRING, NULL, SLAPD_DEFAULT_LDAPI_SEARCH_BASE, NULL},
+    {CONFIG_LDAPI_AUTH_MAP_BASE_ATTRIBUTE, config_set_ldapi_mapping_base_dn,
```

Comment:
Should this have a default value?

Reply:
The ldapi code uses NULL to check if anything is set, so a default "string" is not needed.  Or did you mean something else?

Reply:
I think I'm suggesting that we set a default to cn=auto_bind_map,cn=config here so that admins "don't" need to configure anything beside adding their maps, then IPA can use/tweak this value if they need to customise it. In the current form the maps are "disabled by default" with this always being NULL.